src/projects-card/ProjectsCard.css

```diff
@@ -0,0 +1,23 @@
+.mantine-Card-root {
+    flex-shrink: 0;
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    border-radius: 30px;
+}
+
+.mantine-Card-section {
+    margin: 0;
+}
+
+.project-name {
+    padding-top: 10px;
+    text-align: center;
+    font-size: 55px;
```

Comment:
Just as an addition, you could use responsive units, media queries or clamp/calc css variables so that you get responsive text that also works with smaller screen widths. But since the figma doesn't really specify anything for how the page should look responsively, this is more of an aside thing

<img width="570" alt="image" src="https://github.com/uoftblueprint/website-v2/assets/50971945/3592fb6e-b6bd-43c1-ae7c-af3435b29c93">